packages/react-magma-dom/src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,238 @@
+import * as React from 'react';
+import { ButtonColor, ButtonShape, ButtonSize } from '../Button';
+import { darken } from 'polished';
+import { I18nContext } from '../../i18n';
+import { IconButton } from '../IconButton';
+import { PageButton } from './PageButton';
+import { ArrowBackIcon, ArrowForwardIcon } from 'react-magma-icons';
+import styled from '../../theme/styled';
+import { ThemeContext } from '../../theme/ThemeContext';
+
+export interface PaginationProps extends React.HTMLAttributes<HTMLDivElement> {
+  testId?: string;
+  /**
+   * Size toggles between default and large variant buttons.
+   */
+  size?: PageButtonSize;
+  /**
+   * Count designates the total number of Pagination buttons.
+   */
+  count: number;
+  /**
+   * Default Page allows the user to toggle which Pagination button is active by default.
+   */
+  defaultPage?: number;
+  /**
+   * isInverse allows for inversing the theme pallette.
+   */
+  isInverse?: boolean;
```

Comment:
You can get rid of the description for the `isInverse` prop, because if it's left blank a standard default one gets applied.